common/impl/src/main/java/org/jboss/jca/common/metadata/spec/RaParser.java

```diff
@@ -111,7 +111,7 @@ public Connector parse(XMLStreamReader reader) throws Exception
                break;
             }
             case START_ELEMENT : {
-               if ("2.0".equals(reader.getAttributeValue(null, "version")))
+               if ("2.0".equals(reader.getAttributeValue(null, "version") || "2.1".equals(reader.getAttributeValue(null, "version")))
```

Comment:
there is a typo in here - missing `)` - it won't compile

Reply:
fixed in main